### PR TITLE
Refactor queue

### PR DESCRIPTION
**Pull request by shekhar-cis**
Updated queue according to kivy-mock code

### CODE REVIEW

**Comment by PeterSurda** [comment]
put only refactored queue here, and rebase #1915 on top of this.

**Comment by shekhar-cis** [comment]
> put only refactored queue here, and rebase #1915 on top of this.

Ok

**Review by PeterSurda — APPROVED** [review]

**Comment by PeterSurda** [comment]
rebase

File: src/queues.py [hunk 1]
```diff
@@ -8,7 +8,13 @@
 try:
     from multiqueue import MultiQueue
 except ImportError:
-    from .multiqueue import MultiQueue
+    try:
+        from .multiqueue import MultiQueue
+    except ImportError as e:
+        import sys
+        if 'bitmessagemock' not in sys.modules:
+            raise e
```
Comment (g1itch) [inline src/queues.py:16]: Just raise, e is not needed

File: src/queues.py [hunk 2]
```diff
@@ -8,7 +8,13 @@
...
+        from six.moves.queue import Queue as MultiQueue
```
Comment (g1itch) [inline src/queues.py:17]: MultiQueue = queue.Queue ?

Comment (shekhar-cis) [reply src/queues.py:17]: As discussed with @PeterSurda, we have just monkey patched it for Kivy-mock.

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**